### PR TITLE
D2 examples

### PR DESCRIPTION
**Pull request by dem-evgeny**
Added 2 scripts with out of the box examples to how to run a batch file using D2.
The scripts connected with the documentation:
https://support.demisto.com/hc/en-us/articles/115007336908-Running-batch-with-D2

### CODE REVIEW

**Comment by gshriki** [comment]
why is forcepoint part of this?

**Comment by dem-evgeny** [comment]
Mistake

---------------------
[cid:image001.png@01D2E071.FD485CF0]<https://www.demisto.com/>
Evgeny(Eugene) Palcev
Customer Success
Demisto
+1 415-200-2460

**Comment by gshriki** [comment]
OK. Make sure to fix it, so Forcepoint won't be merged by mistake.

**Comment by dem-evgeny** [comment]
@gshriki Fixed.

File: Scripts/script-D2Execute.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+commonfields:
+  id: ef9edd54-0580-4945-8f06-f43dfb69fb20
+  version: 20
```
Comment (yuvalbenshalom) [inline Scripts/script-D2Execute.yml]: id needs tone the same as name.
version needs to be -1

File: Scripts/script-D2Execute.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  demisto.results(demisto.executeCommand("d2_install", {"system":demisto.args()["name"]}))
+  demisto.results("Running script...")
+  demisto.results(demisto.executeCommand(demisto.args()["scriptName"], {"id":demisto.args()["name"], "using":demisto.args()["name"]}))
+type: python
+tags: []
+enabled: true
+args:
+- name: name
+  required: true
+  default: true
```
Comment (yuvalbenshalom) [inline Scripts/script-D2Execute.yml]: missing system: true

File: Scripts/script-D2Run.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+commonfields:
+  id: 9a18460a-e72f-488a-8112-044c9a7be76a
+  version: 13
```
Comment (yuvalbenshalom) [inline Scripts/script-D2Run.yml]: id needs tone the same as name.
version needs to be -1

File: Scripts/script-D2Run.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  if (env.OS !== 'windows') {
+      throw ('script can only run on Windows');
+    }
+
+  var d2path = pwd();
+  var batch_path = d2path + '\\' + batch_file;
+
+  pack(execute('cmd /c ' + batch_path, 60));
+type: javascript
+tags: []
```
Comment (yuvalbenshalom) [inline Scripts/script-D2Run.yml]: missing system: true

File: Scripts/script-D2Execute.yml (hunk 2 above)
Comment (gshriki) [inline Scripts/script-D2Execute.yml]: not needed any more... done automatically...

File: Scripts/script-D2Run.yml (hunk 4 above)
Comment (gshriki) [inline Scripts/script-D2Run.yml]: not needed any more... done automatically...

**Comment by dorsha** [comment]
@dem-evgeny please fix the code review comments and merge if ready.

**Comment by dem-evgeny** [comment]
Fixed the @yuvalbenshalom requests

**Pull request closed without merging**